### ISSUE TITLE
Add 'Update documentation' step to Meteor release process

### ISSUE THREAD

**Issue by avalanche1** [meteor/meteor#9685]
Reading the release notes for 1.6.1. I've stumbled upon a couple of PRs that improve Meteor API. Went to docs.meter.com to read detailed description and couldn't find any mention of it!
I'm talking about https://github.com/meteor/meteor/pull/9042 and https://github.com/meteor/meteor/pull/9044 for example.
Unfortunately this is not the 1st time I've noticed that pattern.

Therefore I ask Meteor team to add a special step to the release process: 'Update official documentation if needed'.
Also a step 'Update official Guide if needed' would be also nice!

By the way, I'm not sure that all of the public API of Meteor is actually documented in the docs. It would be nice if you added an issue here to inventorize the docs vs actual API. @abernix , ping!
Maybe you have the resource for that task or someone from the community will be willing to do it.

**Comment by abernix** [comment]
We do our best to keep the docs updated, but as with the features themselves, we rely on help from the community to keep everything up to date.

While I agree that proper documentation is imperative to the success of a feature (and for solving its problems), sometimes (not always), its not worth delaying a feature (and the hard work of a contributor) for documentation.  With more hands to help out, this wouldn't be the case.

Just like the Meteor tool itself, the docs are open source and available in their own repository: https://github.com/meteor/docs.  We try (please try to familiarize yourself with the volume of pull-requests and issues we review) to ask contributors of PRs to write the docs when they add the features, but some things fall through the cracks.  As a member of the community who is going through and reviewing things with scrutiny, if you see a PR which doesn't have docs and should have it, please suggest that on the PRs!

Lastly, if you find something missing from the docs, please help us write it, or at the very least, open an issue on the Docs repository (linked above).  We're not secretly hiding the docs away somewhere, we just haven't had time to write them.

There isn't really a tangible action item here, so I'll close this.  Just to summarize though, community help is appreciated and docs are a great way to contribute to the project.

**Comment by avalanche1** [comment]
While I highly appreciate the work you guys are doing @ Meteor, it seems an odd concept to keep the product updated with new features, but keep them undocumented (omg!).
And counting on community in such important job is rather naive.

I've added undocumented features of 1.6.1 (that I've spotted) to the docs repo issue tracker; but I am confident, as before, that you guys at MDG would do well to at least open a new issue in docs issue tracker for each new feature that needs to be documented.
